src/gcp_scanner/test_unit.py

```diff
@@ -612,6 +614,22 @@ def test_pubsub_subs(self):
       )
     )
 
+  def test_cloud_assets(self):
+    """Test CloudAsset list"""
+    self.assertTrue(
+      verify(
```

Comment:
So, the test is failing right now which is expected. Could you please create an empty cloud_assets file similar to other crawlers. I will populate it with actual data for testing afterwards.

Reply:
Oops, missed that. I ran pytest without the file and got the error below (as expected)
![image](https://github.com/google/gcp_scanner/assets/29670290/9e0dfb40-16a7-424b-80e1-ba352585f5e6)

I have made a commit to add the empty file. Now that error is no longer appearing.

Reply:
Ok, I added what's necessary in that file for the check.

Reply:
The test is still failing. Not sure why.

Reply:
Yes, I think the tests would work fine without the `cloud_assets` key and just the list

Reply:
Hi @mshudrak , I updated the `cloud_assets` test file, following which the test passed successfully for me locally. I have commited the change to this branch. Could you rerun the tests and see if they are passing now (for cloud assets)?

Reply:
Thanks for the new change but it is still failing for some reason the output is empty. I tried to debug but do not see any obvious reason of why this is happenning.

Reply:
I see. Maybe there is something happening under the hood which we might be missing. I will try to do some research on my side as well, and share if I get any info. Will close this PR for the time being (and open it in the future if any solution is found). Hope that works :)